packages/client-truth-social/src/base.ts

```
@@ -0,0 +1,150 @@
+import { TruthSocialApi } from './api';
+import { TruthApiConfig, TruthStatus } from './types';
+import { IAgentRuntime, elizaLogger } from '@ai16z/eliza';
+import { RequestQueue } from './utils';
+
+export class ClientBase {
+    public truthApi: TruthSocialApi;
+    public profile: any;
+    public requestQueue: RequestQueue;
+    public lastCheckedPostId: BigInt = BigInt(0);
```

Biome (1.9.4)

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)